src/translations.php

```diff
@@ -59,6 +59,12 @@
         "Longest Streak" => "最長のストリーク",
         "Present" => "今",
     ],
+    "nl" => [
+        "Total Contributions" => "Totale Bijdrage",
+        "Current Streak" => "Huidige Serie",
+        "Longest Streak" => "Langste Serie",
+        "Present" => "Vandaag",
+    ], 
```

Comment:
I'm not sure why the prettier workflow was unable to do it automatically, but I think the issue is the extra space here.
```suggestion
    ],
```